Katello/katello
# Fixes #37934 - Changing wording on bulk host collection add/remove

#### What are the changes introduced in this pull request?
* When adding/removing hosts from a host collection using bulk hc membership we were using the rows selected to say x amount of content hosts instead of the actual count.
* Added a `$scope.ok();` to the add remove actions to close the modal after doing the action so it would not just sit there and have stale data and look bad. Opening up the modal has the correct info then. Rather than trying to hack the table to reload I figured we should just close the modal.

#### What are the testing steps for this pull request?
* Create a host collection and add some hosts to it
* Goto Host Collections -> Details -> Host Collection Membership
* Click add-to or remove and see that it says do you want to remove (1) content hosts even though you have more than 1 content host in the host collections
* Apply PR
* See that the wording does not count anything and just says "Are you sure you want to add/remove the following content hosts from the following host collections"
